### PR TITLE
Issue1617 typo and misc

### LINKED ISSUE #1617: Typo and miscellaneous fixes
This issue is to apply some of the same minor fixes first reported in lbl-srg/modelica-buildings#3016. These changes do not change the model results.
The fixes applied here are:
- `IBPSA.Fluid.Storage.Examples.StratifiedUnloadAtMinimumTemperature`: Variable names and descriptions for the "top", "middle", and "bottom" were mixed. This is corrected.
- `IBPSA.Fluid.Interfaces.PartialTwoPortTransport`: Corrected a typo in documentation.

### PR DESCRIPTION
**Pull request by hcasperfu**
This closes #1617.

### CODE REVIEW

**Comment by mwetter** [comment]
@FWuellhorst : can you please review and approve after which this can be merged to the master branch. Thanks.

File: IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo
Comment (FWuellhorst) [inline IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo:148]: Whitespace should be added after the comma, not after the dot.

File: IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo
Comment (FWuellhorst) [inline IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo:159]: As explained in the review comment. If not wanted, revert. Else, ignore.

**Review by FWuellhorst — APPROVED** [review]
@mwetter 
I don't know how you proceed with minor annotation changes. Most of them don't seem necessary, more like a minor mouse movement in the graphics view or saving the file with different settings in Dymola. In my opinion, if they are not intended, all of them should be reverted. What is the procedure for the IBPSA?

Nevertheless, as the connect statements works in Open Modelica, you may go ahead and merge.

**Comment by mwetter** [comment]
@FWuellhorst : Thanks for the feedback and review
@hcasperfu : Can you please revert the non-needed formatting changes.

File: IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo
Comment (hcasperfu) [reply IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo]: This is fixed in 6ee8e0d.

File: IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo
Comment (hcasperfu) [reply IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo]: These are reverted in 6ee8e0d.

**Comment by hcasperfu** [comment]
> 

I have reverted the irrelevant format changes made by Dymola in 6ee8e0d. I hope I caught all of them.
Should I also make the same changes for the Buildings library?

**Comment by hcasperfu** [comment]
> 

I have reverted the irrelevant format changes made by Dymola in 6ee8e0d. I hope I caught all of them.
Should I also make the same changes for the Buildings library?

**Review by FWuellhorst — APPROVED** [review]
Thanks for the changes, nice work!

**Comment by mwetter** [comment]
@hcasperfu : Thanks for the changes. No need to make them in the Buildings library, I will merge these changes to the Buildings library in the next days.

**Pull request merged**
